### ISSUE TITLE
Cant access admin I dont know where the Support is

### ISSUE THREAD

**Issue by sbbmaster09** [YOURLS/YOURLS#2450]
- [x] Check you are using the [**LATEST** release](https://github.com/YOURLS/YOURLS/releases/latest) or the development branch.
- [x] Make sure you have [**SEARCHED** existing issues](https://github.com/YOURLS/YOURLS/issues?utf8=%E2%9C%93&q=is%3Aissue) first. 
- [x] Read the [**GUIDELINES**](https://github.com/YOURLS/YOURLS/blob/master/.github/CONTRIBUTING.md) linked in the yellow notice box above.
- [x] **COMPLETE** this issue template.

**Describe the bug**
I dont know where to inquire for the support. Can someone help me.
I just uploaded the folder and change the config.php. I am running the /admin to finish the installation but "This site can’t be reached server IP address could not be found."

Am I  following the incorrect way? Please help me thanks

**To Reproduce**
2. Click on '...'
3. Scroll down to '...'

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
- YOURLS version:
  - Plugins enabled:
- PHP version:
- Other relevant details depending on the context (server OS & version, MySQL version, browser version, …)

**Additional context**

**Comment by dgw** [comment]
"Server IP address could not be found" sounds like a DNS problem. You'll want to contact your hosting service.

**Comment by sbbmaster09** [comment]
Hi I tried to re-install now and its this error now:
Incorrect DB config, or could not connect to DB

**Comment by sbbmaster09** [comment]
I followed the instruction in the guidelines.. but got this error. I have this domain2.me inside my hosting server that is domain1.com. I am trying to add yourls inside public_html/domain2.me/ folder. 

the first try I cant connect which says cant reached IP Address, then I re-install again and it success but the result for the domain was domain1.com/domain2.me, I tested the URL shortener but its incorrect because it should only have domain2.me. So I deleted all of it again and re-install.

Now got the Incorrect DB config, or could not connect to DB. What should be my error be? I am so confused. Thanks for your help.
